### DISCUSSION TITLE
When I create a watch for particular search text, the webhook is only called for matches with score 1.0, how to handle situations where my application requires all matches with a threshold >= 0.9?

### DISCUSSION THREAD

**Discussion by pratyooshghildiyal-novo** [moov-io/watchman#510] [Ideas]
My application stores information for all matches which have a score >= 0.9 but webhook only returns matches with a score 1.0. Is there a workaround for this where we can specify the match threshold when we create a watcher for any particular search?

**Comment by adamdecaf** [comment]
We've added some general search improvements landed in https://github.com/moov-io/watchman/pull/511 and https://github.com/moov-io/watchman/pull/512

Watches are likely going away in their current form as well. If you have suggestions please reach out on https://github.com/moov-io/watchman/issues/463
